### PR TITLE
React Redux Capstone

### PR DESCRIPTION
**Pull request by Hector096**
✔ Create Details Page that shows details about each CryptoCurrency
✔ Create a Homepage Page that lists all CryptoCurrency.
✔Create Test for Components
✔ Create test for Reducers

Loom Video:
https://www.loom.com/share/1056ea8486dc4b3094106b469dd80d09

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Switch, Route } from 'react-router-dom';
```
Comment (GzimAsani) [inline src/App.js:1]: - [ ] Please create a `container` folder. Please make a distinction between components and container folders. In containers would be all the components where you manage some state where in components folder should be components used for render only like the header.

File: src/components/CryptoDetail.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```
Comment (GzimAsani) [inline src/components/CryptoDetail.js]: - [ ] Please fix this console error when trying to test your project
Screenshot from 2021-11-20 18-38-29

File: src/__test__/components.test.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { BrowserRouter as Router } from 'react-router-dom';
```
Comment (GzimAsani) [inline src/__test__/components.test.js:1]: - [ ] Since this is a capstone project it would be appreciated to add more than 7 tests.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @Hector096                           
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project :muscle: 

## Required Changes ♻️ 
- [x] Please, add your video presentation in your PR
- [x] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by D4nielJ — APPROVED** [review]
## Hello @Hector096, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**